lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManager.cs

```diff
@@ -118,6 +115,56 @@ public IObserver<T> GetRemoteObserver(IPEndPoint remoteEndpoint)
         }
 
         /// <summary>
+        /// Returns an IRemoteObserver used to send messages to the remote host at
+        /// the specified IPEndpoint.
+        /// </summary>
+        /// <param name="remoteEndpoint">The IPEndpoint of the remote host</param>
+        /// <returns>An IRemoteObserver used to send messages to the remote host</returns>
+        public IRemoteObserver<T> GetUnmanagedRemoteObserver(RemoteEventEndPoint<T> remoteEndpoint)
```

Comment:
@markusweimer  @afchung  Do we expect users to implement IRemoteManager? Or we consider this something that we implement?